### DISCUSSION TITLE
Feature Request: Multi-Block Selection by dragging mouse cursor

### DISCUSSION THREAD

**Discussion by hello-jai** [Krishnagopal-Sinha/better-nothing-glyph-composer#50] [Ideas]
Hey , The app has improved a lot, but selecting blocks one by one is still very difficult .
It would be great to have a feature where users can select multiple blocks by dragging the mouse over them. This would make the process much quicker and more user-friendly.

![Screenshot 2024-09-10 145451](https://github.com/user-attachments/assets/f0bc7c23-dbba-438f-9180-070b69543f02)

**Comment by Krishnagopal-Sinha** [comment]
Hi @hello-jai, this feature has been implemented from v1.1.9 onwards. Present under settings > Enable Drag Select. 
P.S. Do beware that It may get a bit laggy due to how javascript works in general and browser optimizations when you have high number of glyph blocks. I will try to solve this when I get more time though, rest assured.
